docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Predict prices using regression with Model Builder'
+description: This tutorial illustrates how to build a regression model using ML.NET Model Builder to predict prices, specifically, New York City taxi fares.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 06/26/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to predict prices using Model Builder. 
+---
+
+# Predict prices using regression with Model Builder
+
+This tutorial illustrates how to build a [regression model](../resources/glossary.md#regression) using ML.NET Model Builder to predict prices, specifically, New York City taxi fares. The regression machine learning task can be utilized in scenarios where the target or label to predict is numerical.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Prepare and understand the data
+> * Choose a scenario
+> * Load the data
+> * Train the model
+> * Evaluate the model
+> * Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Pre-requisites
+
+For a list of pre-requisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Create a console application
+
+1. Create a **.NET Core Console Application** called "TaxiFarePrediction".
+
+1. Create a directory named *Data* in your project to store the data set files.
+
+1. Install the **Microsoft.ML** NuGet Package:
+
+    In **Solution Explorer**, right-click the *TaxiFarePrediction* project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the **Browse** tab, search for **Microsoft.ML**, select the package in the list, and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed.
+
+## Prepare and understand the data
+
+1. Download the [taxi-fare-train.csv](https://github.com/dotnet/machinelearning/blob/master/test/data/taxi-fare-train.csv) data set and save it to the *Data* folder you've created at the previous step. This data set is used to train and evaluate the machine learning model. This data set is originally from the [NYC TLC Taxi Trip data set](http://www.nyc.gov/html/tlc/html/about/trip_record_data.shtml).
+
+1. In **Solution Explorer**, right-click the *taxi-fare-train.csv* file and select **Properties**. Under **Advanced**, change the value of **Copy to Output Directory** to **Copy if newer**.
+
+Each row in the `taxi-fare-train.csv` data set contains details of trips made by a taxi. 
+
+1. Open the **taxi-fare-train.csv** data set and look at column headers in the first row. Take a look at each of the columns. 
+
+The provided data set contains the following columns:
+
+* **vendor_id:** The ID of the taxi vendor is a feature.
```

Comment:
I think we can indent this whole block into the numbered section. This would improve readability

Reply:
#resolved